zcash_client_backend/src/data_api.rs

```diff
@@ -1514,8 +1514,11 @@ pub trait WalletWrite: WalletRead {
         received_tx: DecryptedTransaction<Self::AccountId>,
     ) -> Result<(), Self::Error>;
 
-    /// Saves information about a transaction that was constructed and sent by the wallet to the
-    /// persistent wallet store.
+    /// Saves information about a transaction constructed by the wallet to the persistent
+    /// wallet store.
+    ///
+    /// The name `store_sent_tx` is somewhat misleading; this must be called *before* the
+    /// transaction is sent to the network.
```

Comment:
This API does not dictate a calling order; while this may be true of the `create_proposed_transaction` this is not a comment is relevant to the database IO capability that this method represents.